### PR TITLE
Remove unsupported Hulu oembed provider

### PR DESCRIPTION
**Pull request by tacitonic**
Added Hulu to 'no longer supported providers' from 'supported providers' list.

Fixes https://github.com/WordPress/gutenberg/issues/23920

Trac ticket: https://core.trac.wordpress.org/ticket/50676
---

### CODE REVIEW

File: src/wp-includes/class-wp-oembed.php [hunk 1]
```diff
@@ -212,6 +210,7 @@ public function __construct() {
 		 * | Photobucket  | photobucket.com      | 2.9.0     | 5.1.0     |
 		 * | Funny or Die | funnyordie.com       | 3.0.0     | 5.1.0     |
 		 * | CollegeHumor | collegehumor.com     | 4.0.0     | 5.3.1     |
+ 		 * | Hulu         | hulu.com		 | 2.9.0     | 5.4.2	 |
```
Comment (talldan) [inline src/wp-includes/class-wp-oembed.php:213]: Hey @tacitonic Thanks again for working on this. 

It looks like the PHP linter that's run on Travis picked up a couple of minor issues related to the white space here:
https://travis-ci.com/github/WordPress/wordpress-develop/jobs/362005117

Looks like the comment should be indented using tabs, but all other spacing within the table should be spaces?

I would maybe try to replicate what's on the other lines 😄

Comment (tacitonic) [reply src/wp-includes/class-wp-oembed.php:213]: I did use both tab as well as space indentation, as only tabs didn't align with the existing pipes. I'll see if I can use tabs exclusively. :)

Comment (tacitonic) [reply src/wp-includes/class-wp-oembed.php]: It seems that spaces were used rather than tabs.

File: src/wp-includes/class-wp-oembed.php [hunk 2]
```diff
@@ -212,6 +210,8 @@ public function __construct() {
 		 * | Photobucket  | photobucket.com      | 2.9.0     | 5.1.0     |
 		 * | Funny or Die | funnyordie.com       | 3.0.0     | 5.1.0     |
 		 * | CollegeHumor | collegehumor.com     | 4.0.0     | 5.3.1     |
+		 * | CollegeHumor | collegehumor.com     | 4.0.0     | 5.3.1     |
```
Comment (Soean) [inline src/wp-includes/class-wp-oembed.php:213]: Now we have CollegeHumor twice in this list. We should remove this line.

Comment (tacitonic) [reply src/wp-includes/class-wp-oembed.php]: Sorry, that went unnoticed 😅

File: src/wp-includes/class-wp-oembed.php [hunk 3]
```diff
@@ -212,6 +210,7 @@ public function __construct() {
 		 * | Photobucket  | photobucket.com      | 2.9.0     | 5.1.0     |
 		 * | Funny or Die | funnyordie.com       | 3.0.0     | 5.1.0     |
 		 * | CollegeHumor | collegehumor.com     | 4.0.0     | 5.3.1     |
+		 * | Hulu         | hulu.com             | 2.9.0     | 5.4.2     |
```
Comment (desrosj) [inline src/wp-includes/class-wp-oembed.php:213]: Just noting that the deprecated version will need to be adjusted when this makes it in. I am debating including this in 5.5. I'll move that conversation to Trac.

**Comment by desrosj** [comment]
Thanks for this, @tacitonic! This was merged into Core in https://core.trac.wordpress.org/changeset/48512.

**Pull request closed without merging**